core/src/main/java/com/amplifyframework/core/model/query/QuerySortBy.java

```diff
@@ -73,19 +99,21 @@ public boolean equals(Object object) {
         }
 
         QuerySortBy that = (QuerySortBy) object;
-        return ObjectsCompat.equals(field, that.field) &&
+        return ObjectsCompat.equals(model, that.model) &&
+                ObjectsCompat.equals(field, that.field) &&
                 ObjectsCompat.equals(sortOrder, that.sortOrder);
     }
 
     @Override
     public int hashCode() {
-        return ObjectsCompat.hash(field, sortOrder);
+        return ObjectsCompat.hash(model, field, sortOrder);
     }
 
     @Override
     public String toString() {
         return "QuerySortBy{" +
-                "field='" + field + '\'' +
+                "model=" + (model == null ? null : Wrap.inSingleQuotes(model)) +
```

Comment:
This is a clever and welcome use of the Wrap utility.

However it looks weird to only use it on one line and not the others.

I would update all lines to use this style, or stick with the existing approach for this new line.

Reply:
the fields on the other lines are `@NonNull`, so I figured there isn't a need to do that.